frontend/app/index.html

```diff
@@ -22,18 +22,40 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.png" />
-    <link rel="preload" href="%PUBLIC_URL%/static/media/SourceSansPro-Regular.0d69e5ff5e92ac64a0c9.woff2" as="font" type="font/woff2" crossorigin>
-    <link rel="preload" href="%PUBLIC_URL%/static/media/SourceSansPro-SemiBold.abed79cd0df1827e18cf.woff2" as="font" type="font/woff2" crossorigin>
-    <link rel="preload" href="%PUBLIC_URL%/static/media/SourceSansPro-Bold.118dea98980e20a81ced.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="shortcut icon" href="/favicon.png" />
+    <link
+      rel="preload"
+      href="./static/media/SourceSansPro-Regular.DZLUzqI4.woff2"
```

Comment:
**question:** Is it possible to generate these `link`s? It's unexpected to me that we have calculated hashes hardcoded in this file. I'd expect something like this to be injected as a build step.

Reply:
I believe so, but it does feel more cumbersome. https://github.com/Applelo/unplugin-inject-preload can inject preloads.

Vite seems to recommend placing the fonts in the public directory `https://vite.dev/guide/assets#the-public-directory`. I think I would lean towards this solution if we want a preference (side benefit is we don't need to change the hashes to make downstream dependencies easier.